extension/init.cmd

```diff
@@ -0,0 +1,20 @@
+@echo off
```

Comment:
is this needed and if so do we need init.sh

Reply:
yes, it's useful both in setting up the repo and in CI